AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.ts

```diff
@@ -113,10 +113,20 @@ export class SummaryComponent implements OnInit, OnDestroy {
         this.bookinConfirmed = this.hearing.status === 'Created';
     }
 
+    setAudioRecordingRequired(currentValue: boolean): boolean {
```

Comment:
as this function purely returns a value, could we maybe name it `isAudioRecordingRequired` instead?

Reply:
I have now made this change.